File: bfn-atlas2/clients/src/main/kotlin/com/bfn/client/data/AccountInfoDTO.java
package com.bfn.client.data;

import net.corda.core.serialization.CordaSerializable;

public class AccountInfoDTO {
    String identifier,
            host,
            name;

    public AccountInfoDTO(String identifier, String host, String name) {
        this.identifier = identifier;
        this.host = host;
        this.name = name;

    }


    public String getIdentifier() {
        return identifier;
    }

    public void setIdentifier(String identifier) {
        this.identifier = identifier;
    }

    public String getHost() {
        return host;
    }

    public void setHost(String host) {
        this.host = host;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: bfn-atlas/clients/src/main/kotlin/com/bfn/client/dto/AccountInfoDTO.kt
package com.bfn.client.dto

class AccountInfoDTO {
    var identifier: String? = null
    var host: String? = null
    var name: String? = null
    var status: String? = null

    constructor(identifier: String?, host: String?, name: String?, status: String?) {
        this.identifier = identifier
        this.host = host
        this.name = name
        this.status = status
    }

    constructor() {}

}
